<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>273</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DiceRoller</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resource.qrc">
    <normaloff>:/logo/logo/WoD.png</normaloff>:/logo/logo/WoD.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox_pool">
            <property name="toolTip">
             <string>The number of dice for this pool without any Modifiers.</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_diceNumber">
            <property name="text">
             <string>&amp;Dice</string>
            </property>
            <property name="buddy">
             <cstring>spinBox_pool</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&amp;Modifier</string>
            </property>
            <property name="buddy">
             <cstring>spinBox_modifier</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinBox_modifier">
            <property name="toolTip">
             <string>The Modifiers for this roll.</string>
            </property>
            <property name="minimum">
             <number>-99</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBox_xAgain">
          <property name="toolTip">
           <string>The reroll-property for this roll.</string>
          </property>
          <item>
           <property name="text">
            <string>10 again</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9 again</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8 again</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>no reroll</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>no reroll &amp; subtract 1s</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_rote">
          <property name="toolTip">
           <string>Activate the Rote-ability (allowing you to reroll all unsuccessfule dice).</string>
          </property>
          <property name="text">
           <string>&amp;Rote-property</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_extended">
        <property name="toolTip">
         <string>Rolls an Extended Roll, gathering successes over multiple rolls of the pool.</string>
        </property>
        <property name="title">
         <string>&amp;Extdended roll</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QRadioButton" name="radioButton_target">
             <property name="text">
              <string>Target Number</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_target">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>The target number for the extended roll.</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QRadioButton" name="radioButton_maxRolls">
             <property name="text">
              <string>maximum Rolls</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_maxRolls">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>The target number for the extended roll.</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_rollsLimited">
           <property name="toolTip">
            <string>Limits the roll to only so many rolls as you have dice in the pool (not counting any modifiers, be they positive or negative).</string>
           </property>
           <property name="text">
            <string>Rolls &amp;Limited</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="pushButton_roll">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Press Button to roll the dice.</string>
        </property>
        <property name="text">
         <string>&amp;Roll</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_result">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/actions/actions/fileclose.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_resultText">
        <property name="text">
         <string>No Result</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_successes">
        <property name="text">
         <string>Successes:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber_successes">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_quit">
        <property name="text">
         <string>&amp;Quit</string>
        </property>
        <property name="icon">
         <iconset resource="resources/resource.qrc">
          <normaloff>:/icons/actions/exit.png</normaloff>:/icons/actions/exit.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>469</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_aboutQt"/>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>E&amp;xtras</string>
    </property>
    <addaction name="action_houserules"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuExtras"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="resources/resource.qrc">
     <normaloff>:/icons/actions/exit.png</normaloff>:/icons/actions/exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the program.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/logo/WoD.png</normaloff>:/icons/logo/WoD.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
   <property name="toolTip">
    <string>About DiceRoller-WoD.</string>
   </property>
  </action>
  <action name="action_houserules">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Houserules</string>
   </property>
   <property name="toolTip">
    <string>The houserule of the programmes group will be used: In Extended rolls, an exceptional success is achieved, whenever any roll during the process shows 5 successes or more.</string>
   </property>
  </action>
  <action name="action_aboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton_roll</tabstop>
  <tabstop>spinBox_pool</tabstop>
  <tabstop>spinBox_modifier</tabstop>
  <tabstop>comboBox_xAgain</tabstop>
  <tabstop>checkBox_rote</tabstop>
  <tabstop>groupBox_extended</tabstop>
  <tabstop>spinBox_maxRolls</tabstop>
  <tabstop>checkBox_rollsLimited</tabstop>
 </tabstops>
 <resources>
  <include location="resources/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_quit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_target</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_target</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_maxRolls</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_maxRolls</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
